virtcontainers/clh.go

```diff
@@ -472,9 +474,43 @@ func (clh *cloudHypervisor) hotplugAddDevice(devInfo interface{}, devType device
 
 }
 
+func (clh *cloudHypervisor) hotplugRemoveBlockDevice(drive *config.BlockDrive) error {
+	cl := clh.client()
+	ctx, cancel := context.WithTimeout(context.Background(), clhHotPlugAPITimeout*time.Second)
+	defer cancel()
+
+	_, _, err := cl.VmmPingGet(ctx)
```

Comment:
Is the ping really needed here?

Reply:
The ping was introduced to quickly check the availability of CLH HTTP server, which allows to bait out quickly when the server is not available.

It is a good point that we may do not want to keep this sanity check in production as it slow down the normal execution path. I think the check is more useful when debugging. Any thoughts? @jcvenegas

Reply:
Yes I would get rid of it, but your call @jcvenegas !

Reply:
Agree with @sboeuf here. Similarly I would get rid of any existing non-required ping invocations in other parts of the code as well in a follow-up PR.